config.yml.example

```
@@ -97,3 +97,15 @@ jail:
     link_influxdb: influxdb
     influxdb_password: unifi-poller
     poller_password: upoller
+
+  grafana:
+    blueprint: grafana
+    ip4_addr: 192.168.1.100/24
+    gateway: 192.168.1.1
+    password: grafana
+    link_influxdb: influxdb
+    database:
+      name: Unifi Controller
+      database: unifi
+      user: unifi
+      password: unifi-poller
```

Needs a white line at the end:
```suggestion
      password: unifi-poller
 
```